### PR TITLE
Manual QA READMEs for trigger, x509, private-endpoint, private-endpoint-adl CFN

### PR DESCRIPTION
**Pull request by maastha**
## Description

Added Manual QA READMEs for trigger, x509, private-endpoint, private-endpoint-adl CFN

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [X] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added any necessary documentation (if appropriate)
- [X] I have run `make fmt` and formatted my code
- [X] For CFN Resources: I have released by changes in the private registry and proved by change works in Atlas

### CODE REVIEW

File: cfn-resources/private-endpoint/test/cfn-test-create-inputs.sh [hunk 1]
```diff
@@ -20,14 +34,20 @@ if [[ "$*" == help ]]; then usage; fi
...
+  region=$AWS_DEFAULT_REGION
 fi
 
-projectName="${1}"
-vpcId="${2}"
-subnetId="${3}"
+#region="${4:-$AWS_DEFAULT_REGION}"
+#if [ -z "$region" ]; then
+#	region=$(aws configure get region)
+#fi
```
Comment (andreaangiolillo) [inline cfn-resources/private-endpoint/test/cfn-test-create-inputs.sh:50]:
```suggestion

```

File: cfn-resources/x509-authentication-database-user/test/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Steps to test:
+1. Follow general [prerequisites](../../../TESTING.md.md#prerequisites) for testing CFN resources.
+2. Follow [general steps](../../../TESTING.md.md#steps) to test CFN resources.
+
+### Success criteria when testing the resource
+1. X.509 Authentication should be enabled correctly set up in your Atlas Project as per configuration specified in the inputs/example. This can be found under Security section -> Advanced:
+
+<img src="https://user-images.githubusercontent.com/122359335/227374480-1afa48a4-5265-4a2a-ad92-067f5015eeca.png">
+2. General [CFN resource success criteria](../../../TESTING.md.md#success-criteria-when-testing-the-resource) should be satisfied.
```
Comment (andreaangiolillo) [inline cfn-resources/x509-authentication-database-user/test/README.md:36]: this link does not seem to be rendered corrently

Comment (maastha) [reply cfn-resources/x509-authentication-database-user/test/README.md:37]: @andreaangiolillo the link for the image?

Comment (andreaangiolillo) [reply cfn-resources/x509-authentication-database-user/test/README.md:37]: this one 

[CFN resource success criteria](../../../TESTING.md.md#success-criteria-when-testing-the-resource)

Comment (andreaangiolillo) [reply cfn-resources/x509-authentication-database-user/test/README.md:37]: Screenshot 2023-03-24 at 10 18 23

Comment (andreaangiolillo) [reply cfn-resources/x509-authentication-database-user/test/README.md:38]: ah, okay nvm my comment then

**Review by andreaangiolillo — APPROVED** [review]

**Pull request merged**